charts/kubescape-operator/values.yaml

```diff
@@ -462,6 +462,9 @@ nodeAgent:
     prometheusExporter: enable
     httpExporterConfig: 
       url: http://synchronizer.kubescape.svc.cluster.local:8089/apis/v1/kubescape.io/v1/runtimealerts
```

Comment:
@amirmalka  I think we should change this so that by default we will export to the synchronizer (when the synchronizer is installed). This way we don't need to deal with the hardcoded namespace, and users will not remove the export from the synchronizer.